Makefile

```diff
@@ -143,6 +143,9 @@ $(TMPTCC)/.git/config:
 $(VC)/.git/config:
 	$(MAKE) fresh_vc
 
+asan:
+	$(MAKE) all CFLAGS='-fsanitize=address,undefined'
```

Comment:
try `CFLAGS='-fsanitize=address,undefined' $(MAKE) all`

Reply:
this will set CFLAGS as an environment variable, unlike the second syntax, which only pass it to make

Reply:
i have simplified the makefile and confirmed that CFLAGS are honored with -showcc before changing this

Reply:
also removed the unnecessary android lines. i dont see why the compiler on android shold generatetd a non executtable program :? it builds and runs fine in termux for me

Reply:
There were reports that it was not executable.
Why are you making so many unrelated changes?

Reply:
If you want to make a cleanup PR, please open another one, so that it can be reviewed and most importantly reverted easily, it there are problems.

Reply:
Sure! Will split this pr into different ones. Any details on how those android builds were done ill try to repro?

Reply:
> Any details on how those android builds were done ill try to repro?

No, unfortunately I did not record the details in the PR that implemented the changes https://github.com/vlang/v/pull/2922 .
I think I remember it was reported on Discord, but can not find it for now (it was in Nov 29, 2019).